src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1748,11 +1736,13 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
         {
             okToDowngradeToNeither = false;
 
-            if (considerRefKinds)
+            var argumentKind = node.Kind;
+            if (considerRefKinds || argumentKind == BoundKind.OutVariablePendingInference || argumentKind == BoundKind.OutDeconstructVarPendingInference)
```

Comment:
> The behavior we want arises from the specification for exactly matching expression.

Please quote what you think the tentative specification is. I am not able to verify correctness of the change otherwise.

---
In reply to: [80793171](https://github.com/dotnet/roslyn/pull/13999#discussion_r80793171) [](ancestors = 80793171,80791006)